src/st/lib/e_shop.c

```diff
@@ -3605,11 +3606,72 @@ void func_us_801B56E4(Entity* self) {
     }
 }
 
-INCLUDE_ASM("st/lib/nonmatchings/e_shop", func_us_801B5F18);
+void func_us_801B5F18(Entity* self) {
+    Entity* tempEntity = &g_Entities[192];
+    CreateEntityFromCurrentEntity(E_ID(ID_25), tempEntity);
+    self->step++;
+#ifdef VERSION_PSP
+    *D_psp_092A54E0 = 0;
+#else
+    D_us_80183F64 = 0;
+#endif
```

Comment:
Is it possible that `D_us_80183F64` is a `u8[4]`? Would both match with an array subscript?